AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -82,8 +82,7 @@ public async Task<ActionResult<HearingDetailsResponse>> Post([FromBody] BookHear
             try
             {
                 var nonJudgeParticipants = newBookingRequest.Participants.Where(p =>
-                        p.CaseRoleName != "Judge" && p.HearingRoleName != "Panel Member" &&
-                        p.HearingRoleName != "Winger")
+                        p.CaseRoleName != "Judge")
```

Comment:
Are these checks not required for 1.27? There are other components which are checking this condition in 

src\app\booking\participant\item\participant-item.component.ts
        return ['Observer', 'Panel Member'].includes(this.participant?.hearing_role_name);